src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,111 @@
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+  const openTodoModal = (todo: Todo) => {
+    setIsOpenModal(true);
+    setActiveTodo(todo);
+  };
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+  const sort = () => {
+    const convertedQuery = query.toLowerCase().trim();
+
+    const todosCopy = [...todos].filter(todo => {
+      return todo.title.toLowerCase().includes(convertedQuery);
+    });
+
+    if (sortBy !== 'all') {
+      return todosCopy.filter(todo => {
+        if (sortBy === 'completed') {
+          return todo.completed === true;
+        }
+
+        return todo.completed === false;
+      });
+    }
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+    return todosCopy;
+  };
+
+  const sortedList = useMemo(() => {
+    return sort();
+  }, [todos, query, sortBy]);
+
+  return (
+    <table className="table is-narrow is-fullwidth">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>
             <span className="icon">
-              <i className="far fa-eye" />
+              <i className="fas fa-check" />
             </span>
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+          </th>
+          <th>Title</th>
+          <th> </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {sortedList.map((todo) => {
+          return (
+            <tr data-cy="todo" className="" key={todo.id}>
+              <td className="is-vcentered">{todo.id}</td>
+              <td className="is-vcentered">
+                {todo.completed && (
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                )}
+              </td>
+              <td className="is-vcentered is-expanded">
+                <p className={`has-text-${!todo.completed ? 'danger' : 'success'}`}>{todo.title}</p>
```

Comment:
use classNames utility
https://www.npmjs.com/package/classnames

Reply:
i think it's not convenient use ternary operator with classnames

Reply:
not fixed